content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Container Deployment"
+metaTitle: "Palette Dev Engine App Profile Container Deployment"
+metaDescription: "Palette Dev Engine App Profile Container Deployment"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Container Deployment
+
+To add a container tier to Palette Dev Engine App Profile,
+
+<br />
+
+1. Log in to the Palette Dev Engine console
+
+
+2. Select the `App Profiles` from the left ribbon menu, click `+ New App Profile` and provide the following information to the app creation wizard.
+
+**Basic Information: **
+
+|         Parameter           | Description  |
+|-------------------------------|-----------------|
+|Application Profile Name | A custom name for the App Profile|
+|Description (optional)   | Description of the App Profile, if any |
+|Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|
+
+
+3. Select **Container Deployment** from the **Services** and provide the following information to the wizard:
+  * **Container Name**: An unique name for the container deployment
+  * **Registry**: Select the registry from which the image is to be extracted
+  * **Image**: Image of the container to be deployed
+  * **Network Access:**
+    * Private
+    * Public
+    * Port number to expose the container for external communication.  
+  * **Environment Variables:** Environment variables can be specified as **Key-Value** pairs during the container deployment.  
+     Example: <br />
+       name: DEMO_GREETING <br />
+       value: Hello from the environment 
+  * **Volume:** To persist the data generated by and used by the container.
+    * Name: Volume name
+    * Size: The size of the volume in GiB
+    * Mount Path:  The path to the volume <br />
+     Example: <br />
+    /var/local/spectro-docs
+
+  * **Runtime Settings**: The command and arguments you define here will override the default command and arguments provided by the container image. <br />
+    Example: <br />
+       command: printenv <br />
+       argument: HOSTNAME
+ 
+ 
+4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
```

Comment:
Change 
```suggestion
4. **Review** the information and complete the app profile creation wizard. Add more tiers to the profile per user requirements, and Palette Clusters can be deployed using the Deploy In A New Palette Virtual Cluster. 
```

Reply:
@resma, tomorrow when you fix the container deployment guide. Let's clean up this paragraph. The "and Palette Clusters can be deployed using the deployed app profile" is awkward and does not make sense.